debian/bcc-tools.install

```diff
@@ -1,3 +1,3 @@
 usr/share/bcc/introspection/*
-usr/share/bcc/tools/*
+usr/share/bcc/tools/*  usr/sbin/
```

Comment:
why not put the new entry, `usr/sbin/` on a new line, like the other entries?

Reply:
this is going to install what is currently installed usr/share/bcc/tools/* in usr/sbin/ with the new shebang